code/script.js

```diff
@@ -1,39 +1,211 @@
+// const botReply = (msg) => {
+//   showMessage(msg, 'bot')
+// }
+
+// const userReply = (msg) => {
+//   showMessage(msg, 'user')
+// }
 
 // Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
-    chat.innerHTML += `
+	if (sender === 'user') {
+		// console.log('is this the user?');
+		chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/dj.png" alt="User" />  
       </section>
-    `
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
+    `;
+	} else if (sender === 'bot') {
+		// console.log('is this the bot?');
+		chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/scuba-diver.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+    `;
+	}
+	// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+	chat.scrollTop = chat.scrollHeight;
+};
+
+// This function clears the content in the innerHTML - used here to clear the input buttons when the chat ends
+const clearInput = () => {
+	inputWrapper.innerHTML = ``;
+};
 
 // Starts here
+// This function will greet the user to the chat
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+	showMessage(`Hello there, are you ready to place your order?`, 'bot');
+  inputWrapper.innerHTML = `
+  <button id="ready">Sure!</button>
+  <button id="notReady">Not ready yet...</button
+  `
+  document.getElementById('ready').addEventListener('click', () => {
+    showMessage(`Sure!`, 'user');
+    startChatting();
+  });
+  document.getElementById('notReady').addEventListener('click', () => {
+    showMessage(`Not ready yet...`, 'user');
+    handleNotReady();
+  });
+}
+
+// This function will get the chat started
+const startChatting = () => {
+  setTimeout(() => {showMessage(`Awesome! Please select the LEGO® theme you're interested in below.`, 'bot');
+  selectTheme();}, 1000);
+}
+
+// This function will handle the not ready reply and ping the user to get started shopping
+const handleNotReady = (message) => {
+  setTimeout(() => {showMessage(`Hello again, are you ready to place your order now?`, 'bot');}, 2000);
+  document.getElementById('notReady').addEventListener('click', () => {
+  });
+}
+
+// This function will present the different LEGO themes available for shopping
```

Comment:
This made me so worried. I really didn't want to go through them all to se each response from the bot. Then I realized quite quickly that it had only two responsens. So clever!